src/Rector/Deprecation/MethodToMethodWithCheckRector.php

```diff
@@ -57,28 +57,41 @@ public function configure(array $configuration): void {
     public function getNodeTypes(): array {
         return [
             Node\Stmt\Expression::class,
+            Node\Expr\MethodCall::class,
         ];
     }
 
     /**
      * @inheritdoc
      */
     public function refactor(Node $node): ?Node {
-        assert($node instanceof Node\Stmt\Expression);
+        assert($node instanceof Node\Stmt\Expression || $node instanceof Node\Expr\MethodCall);
 
-        if (!$node->expr instanceof Node\Expr\MethodCall && !($node->expr instanceof Node\Expr\Assign && $node->expr->expr instanceof Node\Expr\MethodCall)) {
+        if (!$node instanceof Node\Expr\MethodCall && !$node->expr instanceof Node\Expr\MethodCall && !($node->expr instanceof Node\Expr\Assign && $node->expr->expr instanceof Node\Expr\MethodCall)) {
             return NULL;
         }
 
         foreach ($this->configuration as $configuration) {
-            if ($node->expr instanceof Node\Expr\MethodCall && $this->getName($node->expr->name) !== $configuration->getDeprecatedMethodName()) {
+            if ($node instanceof Node\Expr\MethodCall && $this->getName($node->name) !== $configuration->getDeprecatedMethodName()) {
                 continue;
             }
 
-            if ($node->expr instanceof Node\Expr\Assign && $node->expr->expr instanceof Node\Expr\MethodCall && $this->getName($node->expr->expr->name) !== $configuration->getDeprecatedMethodName()) {
+            if ($node instanceof Node\Stmt\Expression && $node->expr instanceof Node\Expr\MethodCall && $this->getName($node->expr->name) !== $configuration->getDeprecatedMethodName()) {
                 continue;
             }
 
+            if ($node instanceof Node\Stmt\Expression && $node->expr instanceof Node\Expr\Assign && $node->expr->expr instanceof Node\Expr\MethodCall && $this->getName($node->expr->expr->name) !== $configuration->getDeprecatedMethodName()) {
+                continue;
+            }
```

Comment:
I feel like this could be optimized to pop out the expressions and find the method call, but that can be done later for this entire rule.